src/core/Room.js

```diff
@@ -80,7 +80,12 @@ class Room {
       data.volatile && data.volatile.sdkInstanceId === this.kuzzle.protocol.id;
 
     if (this.subscribeToSelf || !fromSelf) {
-      this.callback(data);
+      try {
+        this.callback(data);
```

Comment:
A rejected promise returned by the callback will still trigger an unhandledRejection here